### DISCUSSION TITLE
Customization and credit text

### DISCUSSION THREAD

**Discussion by Aerelon** [searx/searx#3345] [General]
I plan on self hosting an instance of Searx, but would like to customize the theme. Is it also possible to remove the "powered by searx" credit? If yes, it's not clear to me how to approach this.

**Comment by kvch** [comment]
You can find the templates here: https://github.com/searx/searx/tree/master/searx/templates/oscar
The text you want to remove is in the base template: https://github.com/searx/searx/blob/master/searx/templates/oscar/base.html
